packages/react-navigation/test/navigation-context.test.tsx

```diff
@@ -1,163 +1,150 @@
-import { type Span } from '@bugsnag/core-performance'
-import { InMemoryDelivery, VALID_API_KEY, createTestClient } from '@bugsnag/js-performance-test-utilities'
-import type BugsnagPerformance from '@bugsnag/react-native-performance'
+import { type SpanFactory } from '@bugsnag/core-performance'
+import { MockSpanFactory } from '@bugsnag/js-performance-test-utilities'
+import { type ReactNativeConfiguration } from '@bugsnag/react-native-performance'
 import { fireEvent, render, screen } from '@testing-library/react-native'
 import React, { useContext } from 'react'
 import { Button, View } from 'react-native'
 import { NavigationContext, NavigationContextProvider } from '../lib/navigation-context'
-import reactNativePlatformExtensions from './utils/react-native-platform-extensions'
```

Comment:
Can this file be deleted now it's no longer being used?

Reply:
💥